### PR TITLE
created dummy community page and controller

### PR DESCRIPTION
**Pull request by AntoineLeclabart**

### CODE REVIEW

File: assets/styles/community.scss [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+        .post-create {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            margin-top: 1.5rem;
+
+            h2 {
+                align-self:flex-start;
+                margin-left: 12vw;
```
Comment (rocambille) [inline assets/styles/community.scss]: looks like you're searching for align-self: center, aren't you?

File: assets/styles/community.scss [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                color: white;
+            }
+
+            input {
+                margin-bottom: 1rem;
+                width: 80vw;
+                height: 10vh;
```
Comment (rocambille) [inline assets/styles/community.scss]: no fixed size

File: assets/styles/community.scss [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                background-color: rgb(117, 221, 172);
+                border: none;
+                border-radius: 25px;
+            }
+
+            button {
+                margin-bottom: 1rem;
+                width: 20vw;
+                height: 2rem;
```
Comment (rocambille) [inline assets/styles/community.scss]: no fixed size

**Review by rocambille — CHANGES_REQUESTED** [review]

File: assets/styles/community.scss (hunk 1 above)
Comment (AntoineLeclabart) [reply assets/styles/community.scss]: no :)

File: assets/styles/community.scss (hunk 2 above)
Comment (AntoineLeclabart) [reply assets/styles/community.scss]: will fix

File: assets/styles/community.scss (hunk 3 above)
Comment (AntoineLeclabart) [reply assets/styles/community.scss]: will fix

Comment (AntoineLeclabart) [reply assets/styles/community.scss]: will fix

**Pull request merged**